src/pixelator/pna/resources/panels/proxiome-immuno-155-v2.csv

```diff
@@ -0,0 +1,167 @@
+# ---
+# name: proxiome-immuno-155-v2
+# aliases:
+#   - proxiome-immuno-155plex-v2
+# description: >-
+#     Panel for the Proxiome Immuno Kit with 155 markers + 3 control markers
+# version: 2.0.0
+# ---
+marker_id,control,full_name,alt_id,uniprot_id,sequence_1,conj_id,sequence_2
+HLA-ABC,,HLA class I histocompatibility antigen,,P04439;P01889;P10321,ACTTCCTAGG,pna_rnd01,ACTTCCTAGG
+B2M,,Beta-2-microglobulin,,P61769,CCAGGTTCCG,pna_rnd02,CCAGGTTCCG
+CD11b,,Integrin α-M (ITGAM),ITGAM;Mac-1 α,P11215,CAGCTATGGT,pna_rnd03,CAGCTATGGT
+CD11c,,Integrin α-X,Integrin αX,P20702,ATATGACGGT,pna_rnd04,ATATGACGGT
+CD18,,Integrin beta 2,Integrin β2,P05107,GGGTAGGCTG,pna_rnd05,GGGTAGGCTG
+CD82,,CD82 antigen,Tetraspanin-27,P27701,CCCGTCATGA,pna_rnd06,CCCGTCATGA
+CD8,,T cell surface glycoprotein CD8 a/b chains,,P01732;P10966,CGGTGGATCA,pna_rnd07,CGGTGGATCA
+TCRab,,T cell receptor alpha chain constant/beta constant 1,,P01848;P01850,CTGGTAGCAA,pna_rnd08,CTGGTAGCAA
+HLA-DR,,"HLA class II histocompatibility antigen, DR",,P01903;P01911,TGTTGTCGAA,pna_rnd09,TGTTGTCGAA
```

Comment:
I think @maxkarlsson also had more detail input on this. But one thing I noticed here is that some of the full names are quoted, and some are not. I am wondering if perhaps we should quote them all? I guess that the reason to have them quoted is that some of them contain `,`?

Reply:
Yes exactly I this is due to the commas. However if we edit this file programmatically in the future, these quotes will be added or removed outside of our control, and it might be hard to enforce quotes on a column.

Since the current file is a valid csv, I would leave it as it is.